eclipse-scout-core/src/util/dates.js

```diff
@@ -424,15 +432,15 @@ export function toJsonDateRange(range) {
  *
  * The format is as follows:
  *
- * [Year#4]-[Month#2]-[Day#2] [Hours#2]:[Minutes#2]:[Seconds#2].[Milliseconds#3][Z]
+ * [Year#4|5]-[Month#2]-[Day#2] [Hours#2]:[Minutes#2]:[Seconds#2].[Milliseconds#3][Z]
```

Comment:
There is an additional function timestamp() that has a similar comment --> check and adjust if necessary

Reply:
I've noticed the function is never used. I'll mark the function as deprecated in 22.0 and remove it in release 23.1